sources/index.php

```diff
@@ -0,0 +1,2 @@
+<?php
+`/usr/local/bin/captiveportal_allow $_SERVER['REMOTE_ADDR']`;
```

Comment:
This files sounds pretty funky x_x .. As far as I understand reading the inital comment of the PR, it's meant to be an example of what YesWiki can do, but if that's the intent then it should be clearly stated in a comment or something